Gruntfile.js

```diff
@@ -136,23 +118,15 @@ module.exports = function( grunt ) {
 	} );
 
 	// Load grunt tasks from NPM packages
-	grunt.loadNpmTasks( "grunt-git-authors" );
-	grunt.loadNpmTasks( "grunt-contrib-concat" );
-	grunt.loadNpmTasks( "grunt-contrib-watch" );
-	grunt.loadNpmTasks( "grunt-contrib-jshint" );
-	grunt.loadNpmTasks( "grunt-jscs" );
-	grunt.loadNpmTasks( "grunt-contrib-uglify" );
-	grunt.loadNpmTasks( "grunt-qunit-istanbul" );
-	grunt.loadNpmTasks( "grunt-coveralls" );
-	grunt.loadNpmTasks( "grunt-npmcopy" );
+	require( "load-grunt-tasks" )( grunt );
 
 	// Integrate jQuery migrate specific tasks
 	grunt.loadTasks( "build/tasks" );
 
 	// Just an alias
 	grunt.registerTask( "test", [ "qunit" ] );
 
-	grunt.registerTask( "lint", [ "jshint", "jscs" ] );
+	grunt.registerTask( "lint", [ "eslint:dev", "eslint:dist" ] );
```

Comment:
This is one task with 2 targets. Not specifying the targets manually would first invoke the `dist` one and then quit; that makes it way harder to know what to fix.